packages/framework/src/Resources/config/services.yaml

```diff
@@ -1118,3 +1118,7 @@ services:
 
     twig.stringloader:
         class: Twig\Loader\ArrayLoader
+
+    Shopsys\FrameworkBundle\Component\Cache\InMemoryCacheResetListener:
+        tags:
+            - { name: doctrine.event_listener, event: onClear }
```

Comment:
This should be properly documented as it may negatively impact performance as resetting all internal caches on `entitymanager->clear()` may not be intuitive.